docs/guides/client.md

```diff
@@ -0,0 +1,83 @@
+# Prism Client
+
+Prism inclues an Axios style HTTP Client that you can use to seamlessy perform requests to both a real server and a mocked document.
```

Comment:
```suggestion
Prism includes an full featured HTTP Client that you can use to seamlessly perform requests to both a real server and a mocked document. The client is modeled after Axios so it may feel familiar.
```

Reply:
I thought it was a good idea to say we inspired to Axios?

Reply:
Ignore me, your suggestion includes that.